spec/system/provider_interface/provider_accepts_data_sharing_agreement.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.feature 'Accept data sharing agreement' do
```

Comment:
Having a single scenario per test file works best for readability and maintainability - I've raised https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1068 to document and discuss.